Testing/functional/tests/cypress/integration/ui/timeline/filter.js

```diff
@@ -133,6 +150,56 @@ describe("Filters", () => {
             "[data-testid=timeline-clinical-document-disclaimer-alert]"
         ).should("not.exist");
     });
+
+    it("Verify diagnostic imaging record alert appears when only imaging reports is selected", () => {
```

Comment:
```suggestion
    it("Verify diagnostic imaging alert appears only when diagnostic imaging is selected", () => {
```

Reply:
The sentences aren't equivalent. its important that the alert only shows when imaging reports filter is selected alone, your sentence version isn't closed to questions like should it show when another filter is selected as well?

imaging reports is the friendly name for the data set.

Reply:
It looks like both the refactored & non-refactored versions of the tests are included currently.

I'm not sure we've used any control structures with test definitions before, so I'd suggest perhaps swapping the `forEach` into a function and calling it in the static tests with parameters passed in from there.

Reply:
![image](https://github.com/bcgov/healthgateway/assets/19548348/ad9e7c7a-5cf5-44cc-b373-ee6c38e2019a)

Reply:
Cleaned that up